components/FlexibleAnswers/FlexibleAnswers.tsx

```diff
@@ -0,0 +1,86 @@
+interface RepeaterGroupAnswer {
+  [key: string]: string | string[];
+}
+
+const RepeaterGroupAnswer = ({
+  data,
+}: {
+  data: RepeaterGroupAnswer;
+}): React.ReactElement => (
+  <ul className="govuk-list lbh-list">
+    {Object.keys(data).map((questionName) => (
+      <li key={questionName}>
+        <strong>{questionName}:</strong>{' '}
+        {Array.isArray(data[questionName])
+          ? (data[questionName] as string[]).join(', ')
+          : data[questionName]}
+      </li>
+    ))}
+  </ul>
+);
+
+const RepeaterGroupAnswers = ({
+  data,
+}: {
+  data: (string | RepeaterGroupAnswer)[];
```

Comment:
We're using `data` a lot as a variable name, could we use something more descriptive?

Reply:
answers?